client/src/docs-ui/menu/menu.style.ts

```diff
@@ -3,8 +3,9 @@ import {css} from "emotion";
 export const menuStyle = css`
   display: block;
   padding: 0 2.5rem;
-  overflow-y: overlay;
-  padding-bottom: 6rem;
+  overflow-y: scroll; /* for Firefox */
```

Comment:
We might want `auto` instead of `scroll` so that it hides the scroll bar if there's no overflow